presidio-api/cmd/presidio-api/api/analyze/analyze_test.go

```diff
@@ -74,3 +74,17 @@ func TestAnalyzeWithNoTemplate(t *testing.T) {
 	assert.Error(t, err)
 
 }
+
+func TestLanguageCode(t *testing.T) {
+
+	api := setupMockServices()
+
+	project := "tests"
+	analyzeAPIRequest := &types.AnalyzeApiRequest{
+		Text:              "My number is (555) 253-0000 and email johnsnow@foo.com",
+		AnalyzeTemplateId: "test",
+		AnalyzeTemplate:   &types.AnalyzeTemplate{},
+	}
+	Analyze(context.Background(), api, analyzeAPIRequest, project)
+	assert.Equal(t, "langtest", analyzeAPIRequest.AnalyzeTemplate.LanguageCode)
```

Comment:
What test cases would cause a failure here? From reading the `Analyze` method the majority of this test is asserting that the `mock` returns the language code expected.

Reply:
@lawrencegripper , since we copy manually files from the proto-gen project,
Failure would occur if the template was not generated as expected. We are testing to verify the existence of the LanguageCode. 
Since no code change were made, but only proto template change were made, we are testing that the change exists.